src/shared/components/Input/Input.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import classNames from 'classnames';
-import './Input.css';
+import './styles/Input.scss';
 
 const Input = ({ classNames: classes, placeholder, onChange, value }) => {
```

Comment:
I can't find prop `classNames` passed to Input. Please check and if it is unnecessary, then remove it